cirq/contrib/noise_models/noise_models.py

```diff
@@ -142,3 +145,121 @@ def noisy_moment(self, moment: 'cirq.Moment',
                 moment,
                 ops.Moment(self.qubit_noise_gate(q) for q in system_qubits)
             ]
+
+
+class PerQubitDepolarizingWithDampedReadoutNoiseModel(devices.NoiseModel):
+    """DepolarizingWithDampedReadoutNoiseModel which allows probabilities to be
+    specified separately for each qubit.
+
+    This simulates asymmetric readout error. The noise is structured
+    so the T1 decay is applied, then the readout bitflip, then measurement.
```

Comment:
Looking at the code, it looks like the actual order of operations is bitflip -> T1 decay -> measurement

Reply:
Whoops. Fixed, and switched to `append` to avoid confusing myself.